### ISSUE TITLE
Out of memory issue java.lang.OutOfMemoryError

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#112]
Here's what I received:

```
java.lang.IllegalStateException Fatal Exception thrown on Scheduler.Worker thread. 
    LooperScheduler.java:114 rx.android.schedulers.LooperScheduler$ScheduledAction.run
    Handler.java:746 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    Looper.java:148 android.os.Looper.loop
    ActivityThread.java:5491 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:728 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:618 com.android.internal.os.ZygoteInit.main


Caused by: java.lang.OutOfMemoryError Failed to allocate a 260997132 byte allocation with 16777216 free bytes and 215MB until OOM 
    VMRuntime.java:-2 dalvik.system.VMRuntime.newNonMovableArray
    Bitmap.java:-2 android.graphics.Bitmap.nativeCreate
    Bitmap.java:857 android.graphics.Bitmap.createBitmap
    Bitmap.java:834 android.graphics.Bitmap.createBitmap
    Bitmap.java:801 android.graphics.Bitmap.createBitmap
    BitmapUtils.java:90 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils.captureView
    BitmapUtils.java:78 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils.captureView
    BitmapUtils.java:39 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils.captureViewWithoutChildren
    BitmapUtils.java:23 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils.access$000
    BitmapUtils.java:29 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils$1.a
    BitmapUtils.java:26 com.instabug.bug.screenshot.viewhierarchy.utilities.BitmapUtils$1.call
    OnSubscribeDefer.java:46 rx.internal.operators.OnSubscribeDefer.call
    OnSubscribeDefer.java:35 rx.internal.operators.OnSubscribeDefer.call
    Observable.java:10150 rx.Observable.unsafeSubscribe
    OperatorSubscribeOn.java:94 rx.internal.operators.OperatorSubscribeOn$1.call
    LooperScheduler.java:107 rx.android.schedulers.LooperScheduler$ScheduledAction.run
    Handler.java:746 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    Looper.java:148 android.os.Looper.loop
    ActivityThread.java:5491 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:728 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:618 com.android.internal.os.ZygoteInit.main
```

Using:
`"react-native": "0.51.0",`
`"instabug-reactnative": "2.1.2",`
Platform: Android

**Comment by ZiadAyman** [comment]
Hey @SudoPlz We are on it!

**Comment by ZiadAyman** [comment]
@SudoPlz Can you confirm with me you are using view hierarchy please?

**Comment by SudoPlz** [comment]
Hey @ZiadAyman , nope I'm not.

This is were how I disable it:

`Instabug.setViewHierarchyEnabled(false); // disabling user view hierarchy`

**Comment by ZiadAyman** [comment]
Let us do some modifications in `Instabug.setViewHierarchyEnabled(false);` and we can try. 
I'll accelerate this as fast as I can and will get back to you with updates!

**Comment by SudoPlz** [comment]
Awesome, I'll be waiting :) thank you.

**Comment by ZiadAyman** [comment]
Hey @SudoPlz 
We fixed this and we released a custom build for you. 
Please `npm install` this branch https://github.com/Instabug/instabug-reactnative#customBuild/disable_view_hierarchy

You might first need to un-link `instabug-reactnative` , install the custom build and then link `instabug-reactnative` back.

Please let me know how is it going!

**Comment by SudoPlz** [comment]
You're the best but I'd love to know when that will make it into the actual build release. I'm not very fond of using custom builds, but if it takes more than a week I might have to.

**Comment by SudoPlz** [comment]
Thanks a ton @ZiadAyman enjoy your weekend :)

**Comment by ZiadAyman** [comment]
It's a pleasure Ioannis :) 
Have a great weekend! @SudoPlz

**Comment by ZiadAyman** [comment]
@SudoPlz Hey man,
This fixed and released on react-native 2.5.0 finally!

**Comment by SudoPlz** [comment]
Amazing, thanks!!! :)
